handlers_test.go

```diff
@@ -11,7 +11,7 @@ import (
 
 const testUUID = "bba39990-c78d-3629-ae83-808c333c6dbc"
 const getTopicsResponse = "[{\"apiUrl\":\"http://localhost:8080/transformers/topics/bba39990-c78d-3629-ae83-808c333c6dbc\"}]\n"
-const getTopicByUUIDResponse = "{\"uuid\":\"bba39990-c78d-3629-ae83-808c333c6dbc\",\"canonicalName\":\"Metals Markets\",\"tmeIdentifier\":\"MTE3-U3ViamVjdHM=\",\"type\":\"Topic\"}\n"
+const getTopicByUUIDResponse = "{\"uuid\":\"bba39990-c78d-3629-ae83-808c333c6dbc\",\"alternativeIdentifiers\":{\"TME\":[\"MTE3-U3ViamVjdHM=\"],\"uuids\":[\"bba39990-c78d-3629-ae83-808c333c6dbc\"]},\"prefLabel\":\"Metals Markets\",\"type\":\"Topic\"}\n"
```

Comment:
I believe you could avoid escaping double quotes by using backticks as your `const` string delimiter.